geth/account/accounts.go

```diff
@@ -218,7 +218,8 @@ func (m *Manager) SelectAccount(address, password string) error {
 		return err
 	}
 
-	if err := whisperService.SelectKeyPair(accountKey.PrivateKey); err != nil {
+	err = whisperService.SelectKeyPair(accountKey.PrivateKey)
+	if err != nil {
```

Comment:
Please, refrain from refactoring in regular issues in the future. Even if you don't like the code. We only do refactoring in refactoring issues to ease merging.

Reply:
There is no refactoring. I've fixed variable shadowing